charts/launch-agent/values.yaml

```diff
@@ -63,6 +63,13 @@ additionalSecretEnvVars:
 # repos. Example: https://username:password@example.com
 gitCreds: |
 
+# list of secrets for the agent to use for ssh auth
+# format is a list of secret names and keys in the format:
+# - name: secret-name
+#   host: example.com
+# secrets should be created ass ssh-auth secrets, see: https://kubernetes.io/docs/concepts/configuration/secret/#ssh-authentication-secrets
+sshAuthSecrets:
```

Comment:
I kinda like the pattern of putting the example values commented below the top key, so that you can just uncomment to get the example. Same for what we do with the other variables above and below